portfolio/pom.xml

```diff
@@ -34,6 +34,12 @@
       <version>2.8.6</version>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
what's this for?

Reply:
guava is so i can convert an iterable to list with as few steps as possible
`entities = Lists.newArrayList(preparedQuery.asIterable());`
This is used in the getXNumOfComments function